plugin/cache/cache.go

```diff
@@ -100,6 +104,17 @@ func hash(qname string, qtype uint16, do bool) uint32 {
 	return h.Sum32()
 }
 
+func computeTTL(msgTTL time.Duration, minTTL time.Duration, maxTTL time.Duration) (ttl time.Duration) {
```

Comment:
you don't need to repeat time.Duration for all these parameters

Reply:
Fixed